source/validate_types.h

```diff
@@ -41,9 +41,107 @@ namespace libspirv {
 // This enum represents the sections of a SPIRV module. The MODULE section
 // contains instructions who's scope spans the entire module. The FUNCTION
 // section includes SPIRV function and function definitions
-enum class ModuleLayoutSection {
-  kModule,    // < Module scope instructions are executed
-  kFunction,  // < Function scope instructions are executed
+// clang-format off
+enum ModuleLayoutSection {
+  kLayoutCapabilities         = 0,  // < Section 2.4 #1
```

Comment:
The initializers (`= 0`) are unnecessary here; the code effect would be identical without them.